pages/task/read/routers/read.js

```diff
@@ -145,31 +145,6 @@ module.exports = () => {
     next();
   });
 
-  app.use((req, res, next) => {
-    const model = req.task.data.model;
-    if (req.task.data.action === 'revoke') {
-      req.task.type = 'revocation';
-      return next();
-    }
-    if (req.task.data.action === 'transfer') {
-      req.task.type = 'transfer';
-      return next();
-    }
-    if (model === 'role' || model === 'profile' || model === 'place') {
-      req.task.type = 'amendment';
-      return next();
-    }
-
-    if (model === 'establishment' && req.task.data.action === 'update') {
-      req.task.type = 'amendment';
-      return next();
-    }
-
-    req.task.type = get(req.task, 'data.modelData.status') === 'active' ? 'amendment' : 'application';
-
-    next();
-  });
```

Comment:
This is all the stuff that got moved to a workflow decorator, right?

Reply:
sure is